README.md

```diff
@@ -49,6 +49,18 @@ Some extensions may specify more specific rules for this field.
 
 This field superceedes the `sar:product_type` field.
 
+#### product:acquisition_type
+
+The product acquisition type in this extension is an enumeration text that providers can use to descibe their product types.
+As described in the [specifications](https://docs.ogc.org/is/10-157r4/10-157r4.html):
+
+> Used to distinguish at a high level the appropriateness of the acquisition for "general" use, whether the product is a nominal acquisition, special calibration product or other.
+
+Admitted values are:
+ - `nominal`
+ - `calibration`
+ - `other`
```

Comment:
I think it would be good to explain these types a bit more and give examples.
I'm not very into this topic myself, so can't really judge whether the enum is extensive enough.

Reply:
Original specification defined in https://docs.ogc.org/is/10-157r4/10-157r4.html, Chapter 8.1.1 EarthObservationMetaData, Table 5, `acquisitionType` Field name, reports:

> Used to distinguish at a high level the appropriateness of the acquisition for "general" use, whether the product is a nominal acquisition, special calibration product or other.
>
>Values:
>      - NOMINAL
>      - CALIBRATION
>      - OTHER

Reply:
That doesn't make it much clearer. As you have a use case for it, I assume you can at least provide examples when these values are to be used?

Reply:
@m-mohr Let me provide this reference: https://sentinel.esa.int/documents/247904/2047089/Sentinel-2_Cal-Val_Phase-E2. Annex A (page 90) provides the calibration sites so some acquisitions over those areas will be acquired for calibration purposes. What this PR brings is the possibility to "flag" products as nominal, calibration or other (not nominal, not calibration). This where my expertise brings us.

Reply:
Great, please summarize them in the extension README.

Reply:
@m-mohr added in https://github.com/stac-extensions/product/pull/5/commits/033c5b924755a316487c109fc88deff4915b310f